routes/lantern.js

```diff
@@ -0,0 +1,149 @@
+const express = require("express");
+const request = require("request");
+const router = express.Router();
+
+const LANTERN_API_URL = "https://api-lantern.ft.com";
+
+const LANTERN_API_KEY = process.env.LANTERN_API_KEY;
+
+router.get("/", (req, res, next) => {
+  res.render("lantern");
+});
+
+router.get("/top-articles/:days", async (req, res, next) => {
+  const dateFrom = new Date();
+  let dateTo = new Date();
+  dateFrom.setDate(dateTo.getDate() - req.params.days);
+
+  const options = {
```

Comment:
The options seem identical for each route, I'm wondering if it would be better to have them as a global const and just add the url to the object for each?